docs/specification/README.md

```diff
@@ -312,6 +312,53 @@ A Data End record indicates the end of the data section.
 | --- | --- | --- | --- |
 | 4 | data_section_crc | uint32 | CRC32 of all bytes from the beginning of the file up to the DataEnd record. A value of 0 indicates the CRC32 is not available. |
 
+### Custom Statistic (op=0x10)
+
+Records some statistic about the MCAP content not covered by the [Statistics](#statistics-op0x0b) record.
+
+| Bytes | Name           | Type    | Description                               |
+| ----- | -------------- | ------- | ----------------------------------------- |
+| 4 + N | name           | string  | the human-readable name of the statistic. |
+| 8     | value          | float64 | The statistic value.                      |
+| 4 + N | subject        | string  | The subject of this statistic.            |
+| 4 + N | display_format | string  | how this statistic should be displayed.   |
+
+#### Custom statistic subjects
+
+Custom Statistics specify a subject, which indicates what this statistic is about. Some subjects are well-known:
```

Comment:
specification of a subject is very similar to what summary offset records do, with their opcode field. it would be ideal if we could use the same mechanism here IMO